### ISSUE TITLE
.local fails on slurm partition

### ISSUE THREAD

**Issue by sjpb** [eth-cscs/reframe#1511]
I have a run-only test I want to run locally:

```python
@rfm.simple_test
class Example(rfm.RunOnlyRegressionTest):
    def __init__(self):
        
        self.valid_systems = ['csd3:cclake-ib-icc19-impi19-ucx']
        self.valid_prog_environs = ['example']
        self.local = True
```

But this gives an error

```python
reframe/bin/reframe: unexpected error: sequence item 0: expected str instance, int found
Traceback (most recent call last):
  File "/home/hpcbras1/hpc-tests/reframe/reframe/frontend/cli.py", line 718, in main
    runner.runall(testcases)
  File "/home/hpcbras1/hpc-tests/reframe/reframe/frontend/executors/__init__.py", line 375, in runall
    self._runall(testcases)
  File "/home/hpcbras1/hpc-tests/reframe/reframe/frontend/executors/__init__.py", line 435, in _runall
    self._policy.exit()
  File "/home/hpcbras1/hpc-tests/reframe/reframe/frontend/executors/policies.py", line 443, in exit
    self._poll_tasks()
  File "/home/hpcbras1/hpc-tests/reframe/reframe/frontend/executors/policies.py", line 358, in _poll_tasks
    *(task.check.job for task in self._running_tasks[partname])
  File "/home/hpcbras1/hpc-tests/reframe/reframe/core/schedulers/slurm.py", line 383, in poll
    f'sacct -S {t_start} -P '
TypeError: sequence item 0: expected str instance, int found
```

This is in a system partition which is configured to use slurm - I thought that was ok and `.local` would override the scheduler setting?

**Comment by vkarak** [comment]
I guess that's on master, right? It should be after we merged #1402.

**Comment by sjpb** [comment]
Thanks, `git pull` fixed it!
